### ISSUE TITLE
Convert to MyST-MD

### ISSUE THREAD

**Issue by samumantha** [coderefinery/train-the-trainer#109]
From PR #105 : 

As part of this we could do a bulk convert to MyST-MD so it looks more normal to people: https://rst-to-myst.readthedocs.io/en/stable/index.html https://docs.readthedocs.io/en/stable/guides/migrate-rest-myst.html

Was this meant for these materials or for all our lesson materials?

Will you do this bulk convert @rkdarst ?

**Comment by bast** [comment]
This material is in MD. And most of our lesson materials are. I think what was meant was possibly sphinx-lesson examples/doc or template repo (but I need to find those first).

**Comment by samumantha** [comment]
So something here: https://github.com/coderefinery/sphinx-lesson and/or here: https://github.com/coderefinery/sphinx-lesson-template ?

**Comment by bast** [comment]
Yes. I think this is what was meant and what would be really good to avoid having to explain and distinguish and switch between RST and MD as we explain and exercise.

**Comment by samumantha** [comment]
I think this was done by @rkdarst; thank you !
